project-reports/2024/2024-Q3-Hyperledger-Indy.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: default
+title: 2024 Q3 Hyperledger Indy
+parent: 2024
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+---
+
+Created by Stephen Curran with input from the Hyperledger Indy Maintainers.
+
+## Sub-Projects
+
+### **Distributed Ledger**
+
+- [indy-node]
+- [indy-plenum]
+- [indy-test-automation]
+- [indy-node-monitor]
+- [indy-node-container]
+- [indy-blssignatures-rs]
+- [indy-besu]
+
+[indy-node]: https://github.com/hyperledger/indy-node
+[indy-plenum]: https://github.com/hyperledger/indy-plenum
+[indy-test-automation]: https://github.com/hyperledger/indy-test-automation
+[indy-node-monitor]: https://github.com/hyperledger/indy-node-monitor
+[indy-node-container]: https://github.com/hyperledger/indy-node-container
+[indy-blssignatures-rs]: https://github.com/hyperledger/indy-blssignatures-rs
+[indy-besu]: https://github.com/hyperledger/indy-besu
+
+### **Client Tools**
+
+- [indy-vdr]
+- [indy-shared-rs] -- for AnonCreds, [Hyperledger AnonCreds] should be used
+- [indy-cli-rs] -- new, Indy CLI based on Indy VDR, not Indy SDK
+
+[indy-vdr]: https://github.com/hyperledger/indy-vdr
+[indy-shared-rs]: https://github.com/hyperledger/indy-shared-rs
+[indy-cli-rs]: https://github.com/hyperledger/indy-cli-rs
+[Hyperledger AnonCreds]: https://github.com/hyperledger/anoncreds-rs
+
+### **Specifications**
+
+- [did:indy Specification], [did:indy Specification source repository]
+- [did:indy Networks]
+- [Indy HIPE] (Hyperledger Indy Project Enhancements)
+- [indy-did-method]
+
+[did:indy Specification]: https://hyperledger.github.io/indy-did-method/
+[did:indy Specification source repository]: https://github.com/hyperledger/indy-did-method
+[did:indy Networks]: https://github.com/hyperledger/indy-did-networks
+[Indy HIPE]: https://github.com/hyperledger/indy-hipe
+[indy-did-method]: https://github.com/hyperledger/indy-did-method
+
+## Project Health
+
+The project had less contributor activity over the quarter. Per the [Indy Quarterly
+Activity Dashboard], in the months April through June, 2024 there were 426
+commits (down about 50%) and 77 contributors (down about 30%). Work was focused
```

Comment:
Q: re: the 30% decrease in contributors, is it due to reduced activity on Indy All less Indy Besu projects or are there other factors to consider??